RecoParticleFlow/PFTracking/src/PFDisplacedVertexFinder.cc

```diff
@@ -734,6 +732,17 @@ PFDisplacedVertexFinder::getTransvDiff(const GlobalPoint& Ref, const GlobalPoint
 
 }
 
+std::pair<float,float>
+PFDisplacedVertexFinder::getTransvLongDiff(const GlobalPoint& Ref, const GlobalPoint& ToProject) const {
+
+  Basic3DVector<float>vRef(Ref);
+  Basic3DVector<float>vToProject(ToProject);
+  double oneOverMag = 1.0/vRef.mag();
+
+  return std::make_pair(fabs(vRef.cross(vToProject).mag()*oneOverMag),fabs((vRef.dot(vToProject)-vRef.mag2())*oneOverMag));
```

Comment:
IMHO, this can all be in float.
all possible essential precision loss can happen in .cross and .dot and .dot-mag2, which all apparently will be made in float.
The extra scaling by 1/mag done in double seems unnecessary.